### PR TITLE
modified origin default

### PR DESCRIPTION
**Pull request by ordirules**
modified the origin of detector, as mentioned in #466 (most recent comment), discussion with @chrisvam, @ericdill 
thanks!

### CODE REVIEW

**Comment by chrisvam** [comment]
Looks good to me.  Thanks, Julien...

File: skbeam/core/accumulators/binned_statistic.py [hunk 1]
```diff
@@ -468,7 +468,7 @@ def __init__(self, shape, bins=10, range=None,
                 represented by function([]), or NaN if this returns an error.
         """
         if origin is None:
-            origin = shape[0]//2, shape[1]//2
+            origin = shape[0]/2., shape[1]/2.
```
Comment (tacaswell) [inline skbeam/core/accumulators/binned_statistic.py]: The doc string says that these should be ints.  Does anything this gets passed into require integers?

Comment (chrisvam) [inline skbeam/core/accumulators/binned_statistic.py]: I believe it's only used to setup the grid, which I believe doesn't require integers.  But you're probably right:  the doc string should be updated.

Comment (ordirules) [inline skbeam/core/accumulators/binned_statistic.py]: updated, @tacaswell thanks for catching that, @chrisvam thanks for input

**Comment by ordirules** [comment]
thanks. test has been updated to test for `origin=None` option as well. Also had to make one of the dimensions odd to properly test, and recompute the `rgrid` and `phigrid` in the loop since origin is modified in the kwargs.

**Pull request merged**